src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -318,7 +318,7 @@ internal override Task<CompletionDescription> GetDescriptionWorkerAsync(Document
         private static INamedTypeSymbol? TryGetTypeWithStaticMembers(ITypeSymbol type)
         {
             if (type.TypeKind is TypeKind.Struct or TypeKind.Class)
-                return type as INamedTypeSymbol;
+                return type.WithNullableAnnotation(NullableAnnotation.NotAnnotated) as INamedTypeSymbol;
```

Comment:
i'm fine with this. but i think a comment explaining the 'why' would be appropriate.